pkg/network/apis/network/types.go

```diff
@@ -17,12 +17,18 @@ type ClusterNetwork struct {
 	metav1.TypeMeta
 	metav1.ObjectMeta
 
+	ClusterNetworks  []ClusterNetworkEntry
 	Network          string
 	HostSubnetLength uint32
```

Comment:
We need to keep old Network and HostSubnetLength fields for backward compatibility? (support config written by 3.7 openshift master and read by 3.6 openshift master) If yes, as part of conversions did we populate these values with ClusterNetworks[0]?

Reply:
No, these values are unpopulated. They are only used in the sdn and I removed all use of Network and HostSubnetLength (except for validating if manually created that the values are valid)

Reply:
Just got confirmation from Clayton that we support config written by 3.7 master to be read by 3.6 master. To allow this, we also need to add ClusterNetworks[0] to DeprecatedNetwork/DeprecatedHostSubnetLength conversion.

Reply:
@JacobTanenbaum Ignore my previous comment, we are not changing user config and we should not change user config automatically with ansible. Field to List conversion is good and we don't need the reverse conversion.